### PR TITLE
Add filter for enabling native lazyload

### PR DESCRIPTION
**Pull request by Tabrisrp**
This change is to be in sync with what we are doing in WP Rocket: native lazyload disabled by default, enabled with a filter

### CODE REVIEW

File: src/Subscriber/LazyloadSubscriber.php [hunk 1]
```diff
@@ -144,19 +144,29 @@ public function insertLazyloadScript()
 
         $inline_args = [
             'threshold' => $threshold,
-            'options'   => [
-                'use_native' => 'true',
-            ],
         ];
 
+        if (apply_filters('rocket_use_native_lazyload', false)) {
```
Comment (Screenfeed) [inline src/Subscriber/LazyloadSubscriber.php]: Here too, missing DocBlock.

File: src/Subscriber/LazyloadSubscriber.php [hunk 2]
```diff
@@ -144,19 +144,29 @@ public function insertLazyloadScript()
...
+            $inline_args['options'] = [
+                'use_native' => 'true',
+            ];
+        }
+
         if ($this->option_array->get('images') || $this->option_array->get('iframes')) {
-            $inline_args['elements']['loading'] = '[loading=lazy]';
+            if (apply_filters('rocket_use_native_lazyload', false)) {
```
Comment (Screenfeed) [inline src/Subscriber/LazyloadSubscriber.php]: And... here.

**Comment by Screenfeed** [comment]
Strange, is `// phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals` required when the hook is prefixed with `rocket_`? 🤔

**Comment by Tabrisrp** [comment]
The prefix defined in the phpcs file is `rocket_lazyload`, that's why

**Pull request merged**